src/Before.Orleans.sln.targets

```diff
@@ -42,7 +42,7 @@
 
                     Log.LogMessage("Downloading latest version of NuGet.exe...");
                     WebClient webClient = new WebClient();
-                    webClient.DownloadFile("https://www.nuget.org/nuget.exe", TargetPath);
+                    webClient.DownloadFile("https://dist.nuget.org/win-x86-commandline/latest/nuget.exe", TargetPath);
 
```

Comment:
@attilah where did you get this new URL from? The [official documentation](http://docs.nuget.org/consume/command-line-reference) says you should download from the previous location.